### PR TITLE
New feature - serviceManagement added to handle rtk types and methods

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/components/shared/basic/ReleaseProcess/OfferCard/index.tsx [hunk 1]
```diff
@@ -211,7 +195,8 @@ export default function OfferCard() {
       .unwrap()
       .then(() => {
         dispatch(setServiceId(serviceId))
-        buttonLabel === 'saveAndProceed' && dispatch(increment())
+        buttonLabel === 'saveAndProceed' &&
```
Comment (oyo) [inline cx-portal/src/components/shared/basic/ReleaseProcess/OfferCard/index.tsx:198]: was there before but please fix and create a const or enum type for this

File: cx-portal/src/components/shared/basic/ReleaseProcess/OfferCard/index.tsx [hunk 2]
```diff
@@ -224,40 +209,18 @@ export default function OfferCard() {
...
-              setFileStatus(UploadStatus.UPLOAD_SUCCESS)
-            })
-            .catch(() => {
-              setFileStatus(UploadStatus.UPLOAD_ERROR)
-            })
-
+          //TO-DO Image file upload
           dispatch(setServiceId(result))
-          buttonLabel === 'saveAndProceed' && dispatch(increment())
+          buttonLabel === 'saveAndProceed' &&
```
Comment (oyo) [inline cx-portal/src/components/shared/basic/ReleaseProcess/OfferCard/index.tsx:221]: please convert to const or enum type

File: cx-portal/src/components/shared/basic/ReleaseProcess/OfferPage/index.tsx [hunk 3]
```diff
@@ -218,7 +218,8 @@ export default function OfferPage() {
         id: serviceId,
         body: apiBody,
       }).unwrap()
-      buttonLabel === 'saveAndProceed' && dispatch(increment())
+      buttonLabel === 'saveAndProceed' &&
```
Comment (oyo) [inline cx-portal/src/components/shared/basic/ReleaseProcess/OfferPage/index.tsx:221]: const or enum type

File: cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonContractAndConsent.tsx [hunk 4]
```diff
@@ -305,7 +310,14 @@ export default function CommonContractAndConsent({
       await updateAgreementConsents(updateData)
         .unwrap()
         .then(() => {
-          buttonLabel === 'saveAndProceed' && dispatch(increment())
+          if (
+            buttonLabel === 'saveAndProceed' &&
```
Comment (oyo) [inline cx-portal/src/components/shared/basic/ReleaseProcess/components/CommonContractAndConsent.tsx:314]: const or enum type

**Review by oyo — CHANGES_REQUESTED** [review]
please convert the hard coded string to a const or enum type

**Comment by jjeroch** [comment]
@manojava-gk  can you please check the pr name. It will be tough for anyone to know what is inside. I suggest to update the name to something more meaningful and keep the connection to the "Service Release Process"

**Review by oyo — APPROVED** [review]
looks good now :)

**Pull request merged**